### PR TITLE
implemented account menu with header

### PR DESCRIPTION
**Pull request by Mav-Ivan**
Tests editProfile and SidebarMenu will be for newcomers

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110425368/0f47b347-3c94-4caa-8795-7d45cc9f94a9

### CODE REVIEW

File: src/pages/edit-profile/EditProfile.constants.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <h4>Profile</h4>
+      </Box>
+    )
+  },
+  [TutorProfileTabsEnum.Notifications]: {
+    icon: <NotificationsIcon />,
+    title: 'editTutor.main.notifications',
+    content: (
+      <Box>
+        <h4>Notifications</h4>
```
Comment (dmtrth25) [inline src/pages/edit-profile/EditProfile.constants.tsx:32]: also use `Typography` for all `Box` with `h4` `enum` variant

File: src/pages/edit-profile/EditProfile.constants.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </Box>
+    )
+  },
+  [TutorProfileTabsEnum.PasswordAndSecurity]: {
+    icon: <GppGoodIcon />,
+    title: 'editTutor.main.passwordSecurity',
+    content: (
+      <Box>
+        <h4>Password & Security</h4>
```
Comment (dmtrth25) [inline src/pages/edit-profile/EditProfile.constants.tsx:41]: the same for `h4` 
and don't forget about pass your translations

File: src/components/sidebar-menu/SidebarMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { TutorProfileProps } from '~/pages/edit-profile/EditProfile.constants'
+import { TutorProfileTabsEnum } from '~/types'
+
+interface SidebarMenu {
+  tabsData: TutorProfileProps
+  handleClick: (tab: TutorProfileTabsEnum) => void
+  styles?: SxProps<Theme>
+}
+
+const SidebarMenu = ({ handleClick, tabsData, styles }: SidebarMenu) => {
```
Comment (dmtrth25) [inline src/components/sidebar-menu/SidebarMenu.tsx:21]: I think we use `FC` and pass there generic for types. What do you think ?

File: src/pages/edit-profile/EditProfile.tsx [hunk 4]
```diff
@@ -1,9 +1,58 @@
...
+import Divider from '@mui/material/Divider'
+
 import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import AppButton from '~/components/app-button/AppButton'
+import SidebarMenu from '~/components/sidebar-menu/SidebarMenu'
+import { ButtonVariantEnum, SizeEnum } from '~/types'
+import { tabsData } from '~/pages/edit-profile/EditProfile.constants'
+import { TutorProfileTabsEnum } from '~/types'
+
+import { styles } from './EditProfile.styles'
```
Comment (dmtrth25) [inline src/pages/edit-profile/EditProfile.tsx:15]: use aliases here

Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.tsx]: done

File: src/components/sidebar-menu/SidebarMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import {
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  List,
+  SxProps,
+  Theme
```
Comment (Tolik170) [inline src/components/sidebar-menu/SidebarMenu.tsx:8]: import Each component directly to reduce bundle size

File: src/pages/edit-profile/EditProfile.constants.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  }
+}
+
+export const tabsData: TutorProfileProps = {
+  [TutorProfileTabsEnum.Profile]: {
+    icon: <AccountCircleIcon />,
+    title: 'editTutor.main.profile',
+    content: (
+      <Box>
+        <h4>Profile</h4>
```
[the remark this answers is not in the document]
Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.constants.tsx:23]: No need it's just a dummy text. There will be some content for that tab

File: src/constants/translations/ua/edit-tutor-profile.json [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
+    "main":{
+        "accountSettings": "Налаштування Аккаунта",
+        "littleDescription": "Керуйте параметрами свого облікового запису",
+        "backBtn":"Повернутися до особистого профілю",
+        "profile": "Профіль",
+        "notifications":"Сповіщення",
+        "passwordSecurity": "Пароль і Безпека"
+    }
+}
```
[the remark this answers is not in the document]
Comment (Mav-Ivan) [reply src/constants/translations/ua/edit-tutor-profile.json]: Sure

File: src/pages/edit-profile/EditProfile.constants.tsx [hunk 8]
```diff
@@ -0,0 +1,45 @@
+import { ReactElement } from 'react'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle'
+import NotificationsIcon from '@mui/icons-material/Notifications'
+import GppGoodIcon from '@mui/icons-material/GppGood'
+import { Box } from '@mui/system'
```
Comment (Tolik170) [inline src/pages/edit-profile/EditProfile.constants.tsx:5]: import Box from '@mui/material/Box'

File: src/pages/edit-profile/EditProfile.constants.tsx (hunk 1 above)
Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.constants.tsx:32]: No need it's just a dummy text. There will be some content for that tab

File: src/pages/edit-profile/EditProfile.constants.tsx (hunk 2 above)
Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.constants.tsx:41]: No need it's just a dummy text. There will be some content for that tab

File: src/components/sidebar-menu/SidebarMenu.tsx (hunk 3 above)
Comment (Mav-Ivan) [reply src/components/sidebar-menu/SidebarMenu.tsx]: Agree with you

File: src/pages/edit-profile/EditProfile.tsx [hunk 9]
```diff
@@ -1,9 +1,58 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Typography } from '@mui/material'
```
Comment (Tolik170) [inline src/pages/edit-profile/EditProfile.tsx:4]: import Typography from '@mui/material/Typography '

File: src/pages/edit-profile/EditProfile.constants.tsx (hunk 1 above)
Comment (dmtrth25) [reply src/pages/edit-profile/EditProfile.constants.tsx:32]: > No need it's just a dummy text. There will be some content for that tab

ok. it's like mocked text there for future tabs content

File: src/components/sidebar-menu/SidebarMenu.tsx (hunk 5 above)
Comment (Mav-Ivan) [reply src/components/sidebar-menu/SidebarMenu.tsx]: Ok, done

File: src/pages/edit-profile/EditProfile.constants.tsx (hunk 8 above)
Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.constants.tsx]: done

File: src/pages/edit-profile/EditProfile.tsx (hunk 9 above)
Comment (Mav-Ivan) [reply src/pages/edit-profile/EditProfile.tsx]: done

**Review by dmtrth25 — APPROVED** [review]

**Review by abalanovsky — APPROVED** [review]

**Review by Mygal-Alex — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**